neo/Prompt/Commands/tests/test_wallet_commands.py

```diff
@@ -240,6 +257,61 @@ def test_wallet_create_address(self):
         self.assertEqual(type(res), UserWallet)
         self.assertEqual(len(res.Addresses), 9)
 
+    def test_wallet_claim_1(self):
+        # test with no wallet
+        args = ['claim']
+        res = CommandWallet().execute(args)
+        self.assertFalse(res)
+
+        self.OpenWallet1()
+
+        # test with max_coins_to_claim non integer
+        args = ['claim', 'abc']
+        claim_tx, relayed = CommandWallet().execute(args)
+        self.assertEqual(claim_tx, None)
+        self.assertFalse(relayed)
+
+        # test with max_coins_to_claim <= 0
+        args = ['claim', '-1']
+        claim_tx, relayed = CommandWallet().execute(args)
+        self.assertEqual(claim_tx, None)
+        self.assertFalse(relayed)
+
+        # test wrong password
+        with patch('neo.Prompt.Commands.Wallet.prompt', side_effect=["wrong"]):
+            args = ['claim']
+            claim_tx, relayed = CommandWallet().execute(args)
+            self.assertEqual(claim_tx, None)
+            self.assertFalse(relayed)
+
+        # test successfull
+        with patch('neo.Prompt.Commands.Wallet.prompt', side_effect=[WalletFixtureTestCase.wallet_1_pass()]):
+            args = ['claim']
+            claim_tx, relayed = CommandWallet().execute(args)
+            self.assertIsInstance(claim_tx, ClaimTransaction)
+            # self.assertTrue(relayed)
```

Comment:
`relayed` is always false here, probably because of `neo/Prompt/Commands/tests/test_claim_command.py`.
How should I generate new wallets with GAS to claim for test?
Please tell me if you have another idea.

Reply:
I don't think that `test_claim_command.py` plays a role here. Reason being that your testcase `UserWalletTestCase` extends `WalletFixtureTestCase` which in turn extends `BlockchainFixtureTestCase`. 
- `BlockchainFixtureTestCase` makes it so that each run we reset the blockchain state. Meaning any transactions / changes we performed on the blockchain in a test get reset.
- `WalletFixtureTestCase` adds to this that each wallet gets copied when running the test to make sure we never modify the original wallet. As with the blockchain, this ensures that we always have the same starting state for wallets when running a test case.

Reply:
My first check would be to validate that the wallets you're using actually have unclaimed GAS. Although it should looking the `test_claim_command.py`. All I can suggest is stepping through the code because that's what I would have to do as well to debug. I'm guessing this fails:
https://github.com/CityOfZion/neo-python/blob/e3780126f9cf8175f9f3c7ad27f36f2baae20508/neo/Network/NodeLeader.py#L323

Reply:
Oh that's important information. In that case I'd probably look at which tests use `WalletFixtureTestCase` and disable those tests completely to verify if any of them mess up the state (for example because they do not copy the wallet files). 

Note that the overall tests fail due to a pylinting error not because a test is failing (although that might make sense since you commented parts out). See: https://travis-ci.org/CityOfZion/neo-python/builds/464202907#L3226

Reply:
I saw this, I will fix it :sweat_smile: 
The parts I commented out fail in local when running `make test`.

I will investigate, thank you for your directions!

Reply:
I opened the PR #742